src/models/axios-error-event-decorator.ts

```diff
@@ -27,7 +27,7 @@ export class AxiosErrorEventDecorator {
 		return this.response?.request;
 	}
 
-	static decorate(event: any, hint: any): any {
+	static decorate(this: void, event: any, hint: any): any {
```

Comment:
/nit /opinion-piece lol - From what I understand, the `this: void` is just another way of saying this is a static method? Typescript will already enforce rules around calling this function with the `static` declaration so seems just like extra effort having  `this: void`

Reply:
This is related to this eslint rule https://typescript-eslint.io/rules/unbound-method/

My understanding of it is when passing this function into a method you don't get the class binding.  Adding this to the function ensures you don't try to access anything internally on the class in the method call.